storage/database/dynamodb.go

```diff
@@ -125,25 +131,38 @@ func NewDynamoDB(config *DynamoDBConfig) (*dynamoDB, error) {
 		config.Endpoint = "https://dynamodb." + config.Region + ".amazonaws.com"
 	}
 
-	logger.Info("creating s3FileDB ", "bucket", config.TableName)
+	dynamoOpenedDBNum++
```

Comment:
This line should be placed right after this function is successfully finished.

Reply:
review reflected :) Thanks!